facilitator/taplo.toml

```diff
@@ -0,0 +1,4 @@
+include = ["Cargo.toml"]
```

Comment:
What does this do?

Reply:
Was thinking we can potentially use https://taplo.tamasfe.dev/ to keep our `Cargo.toml` files in proper formatting. Was just experimenting with it and won't include it in this PR at the end. Will make a separate PR for it later.